tests_aws/conftest.py

```diff
@@ -1,15 +1,16 @@
-# (c) Copyright IBM Corp. 2021
-# (c) Copyright Instana Inc. 2020
+# (c) Copyright IBM Corp. 2024
 
 import os
 import platform
 
 os.environ["INSTANA_ENDPOINT_URL"] = "https://localhost/notreal"
 os.environ["INSTANA_AGENT_KEY"] = "Fake_Key"
 
-# ppc64le is not supported by AWS Serverless Services.
+# ppc64le and s390x are not supported by AWS Serverless Services.
 collect_ignore_glob = []
-if platform.machine() == "ppc64le":
-    collect_ignore_glob.append("*test_lambda*")
-    collect_ignore_glob.append("*test_fargate*")
-    collect_ignore_glob.append("*test_eks*")
\ No newline at end of file
+if platform.machine() in ["ppc64le", "s390x"]:
+    collect_ignore_glob.extend([
+        "*test_lambda*",
+        "*test_fargate*",
+        "*test_eks*",
+    ])
```

Comment:
Please add a new line at EOF
```suggestion
    ])
    
```